hw/ip/otp_ctrl/doc/dv/index.md

```diff
@@ -62,23 +62,25 @@ All common types and methods defined at the package level can be found in
 
 ### TL_agent
 OTP_CTRL testbench instantiates (already handled in CIP base env) [tl_agent]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
-which provides the ability to drive and independently monitor random traffic via
-TL host interface into OTP_CTRL device.
+which provides the ability to drive and independently monitor random traffic via TL host interface into OTP_CTRL device.
 
-### Alert agents
+### Alert_agents
 OTP_CTRL testbench instantiates (already handled in CIP base env) two [alert_agents]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}):
 fatal_check_alert and fatal_macro_alert.
-The alert_agents provide the ability to drive and independently monitor alert handshakes via
-alert interfaces in OTP_CTRL device.
+The alert_agents provide the ability to drive and independently monitor alert handshakes via alert interfaces in OTP_CTRL device.
 
-### OTP_CTRL IF
+### OTP_CTRL interface
 OTP_CTRL design has specific inputs and outputs to communicate with other IPs including LC_CTRL, OTBN, SRAM, FLASH etc.
 This interface is created to initialize, use simple task to drive, and use assertions to monitor these signals.
 
-### UVM RAL Model
+### Memory backdoor interface
+OTP_CTRL testbench binds design's non-volatile OTP memory with a [`mem_bkdr_if`]({{< relref "hw/dv/sv/mem_bkdr_if/README.md" >}}),
```

Comment:
small nit: one sentence per line

Reply:
let me update the template as well with a separate PR :) Thanks for pointing that out.